spec/models/affiliate_spec.rb

```diff
@@ -76,35 +76,60 @@
     it { is_expected.to validate_inclusion_of(:search_engine).in_array(%w( Google BingV6 BingV7 SearchGov )) }
 
     it { is_expected.to have_many :boosted_contents }
+    it { is_expected.to have_many(:connections).inverse_of(:affiliate) }
+    it { is_expected.to have_many(:connected_connections).inverse_of(:affiliate) }
```

Comment:
Should be `inverse_of(:connected_affiliate)`

Reply:
I'm guessing you mean the test line below that which is testing  ` it { is_expected.to have_many(:connected_connections).inverse_of(:connected_affiliate) }`